firmware/application/apps/ui_playlist.cpp

```diff
@@ -193,20 +193,9 @@ void PlaylistView::start() {
             });
     }
 
-    // Enable Bias Tee if selected
-    radio::set_antenna_bias(portapack::get_antenna_bias());
-
-    rf_amp = (transmitter_model.rf_amp());  // recover rf_amp settings applied from ui_transmiter.cpp
-
-    radio::enable({receiver_model.tuning_frequency(),
-                   sample_rate * 8,
-                   baseband_bandwidth,
-                   rf::Direction::Transmit,
-                   rf_amp,  // previous code line : "receiver_model.rf_amp()," was passing the same rf_amp of all Receiver Apps
-                   static_cast<int8_t>(receiver_model.lna()),
-                   static_cast<int8_t>(receiver_model.vga())});
-
-    //	}
+    transmitter_model.set_sampling_rate(sample_rate * 8);
+    transmitter_model.set_baseband_bandwidth(baseband_bandwidth);
+    transmitter_model.enable();
```

Comment:
Is it making a call to transmitter_model.rf_amp() inside enable() ?
If not then it's killing the lna and vga set from the receiver side

Reply:
It uses whatever value is set in the transmitter_model. Which would be either the default or whatever was set via TransmitterView.

Reply:
But you’re right, the previous code was using the receiver values. Good catch.

Reply:
Basically  I also  agree with those changes,  but we are changing so many files,  so we need to check carefully the binary.
But you are deleting some of my previous  comments , that was making  easy to follow before ...  But anyway .

So  if I can get  beta binary in the Test drive,  I can also  check  it .
Thanks ,  the code looks like more  tidy up . 
Cheers.

Reply:
There is a build in Test-a drive if you want to try it. I deleted the bias tee comments because it shouldn’t be a concern at the app layer.